### PR TITLE
Adding chlorophyll and height data to master

### PR DESCRIPTION
**Pull request by readersm**
I made sure all of the column headings were consistent with what Daniel added. I noticed that a few things such as 'CropName' are not in the README file so I updated that. 

I also am not sure what units the CHL meter you all are using outputs, so I have kept my column heading separate until we figure that out, but I can change that later. I also added the column headers for those to the README.md file.

I have CHL and heights together for the first and second seasons (so you will notice some NAs where I didn't have enough plants to do some of the measurements some weeks. There are no empty cells though so filtering them out in R should be easy. 

I kept the CHL and heights separate for the third season as I saw Daniel has them separated (in case that's the route we want to take.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@
 `biomass_soil_[season]_[crop].csv` (`biomass_soil_winter_SH.csv`)
 - `Location`: experimental plot location at TREC [*linking field*]
 - `CropTrt`: experimental treatment denoting cover crops in monoculture or mixture [*linking field*]
+- 'CropName': abbreviated name of individual cover crops e.g. SH for sunn hemp
```
Comment (brymz) [inline README.md]: These quotes `'` should be backticks `` ` `` .

File: data/chl-summer-2019-ECHO.csv [hunk 2]
```diff
@@ -0,0 +1 @@
...
176,VB,8,7,9,2019,37.5,27,21,92.2
176,VB,8,7,9,2019,35,24.5,18.2,92
178,SS,8,7,9,2019,30.8,20.4,13.6,91.8
178,SS,8,7,9,2019,21.8,11.5,4.8,91.8
178,SS,8,7,9,2019,29.1,18.7,11.8,91.8
179,SS,8,7,9,2019,30.6,20.2,13.4,91.8
179,SS,8,7,9,2019,20.2,9.9,3.4,91.8
179,SS,8,7,9,2019,31.9,21.5,14.8,91.7
180,VB,8,7,9,2019,27.8,17.4,10.5,91.6
180,VB,8,7,9,2019,46.1,35.5,31.7,91.5
```
Comment (brymz) [inline data/chl-summer-2019-ECHO.csv]: This file came out as only one line.

File: README.md (hunk 1 above)
Comment (readersm) [reply README.md]: oops! I just fixed that. Took me a while to figure out where the back ticks were haha

File: data/chl-summer-2019-ECHO.csv (hunk 2 above)
Comment (readersm) [reply data/chl-summer-2019-ECHO.csv]: Fixed. apparently the csv that get's downloaded from the CHL metering device needed re-exporting to a csv. Good to know!

**Pull request merged**